03_008/src/pages/Home/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ContentList } from '@/components';
+import { Ranking, RecentContent } from '@/containers';
+import { createElement } from '@/core/dom';
+import { FunctionComponent } from '@/types/dom';
+import homeStyle from './home.module.scss';
+
+interface HomeProps {}
+
+const { home } = homeStyle;
+
+export const Home: FunctionComponent<HomeProps> = () => {
+  return createElement(`<main class="${home}"></main>`, [RecentContent({}), Ranking({})]);
```

Comment:
```suggestion
  return createElement(`
    <main class="${home}">
      ${RecentContent({})}
      ${Ranking({})}
    </main>
  `);
```
앞선 리뷰의 연장선으로, 이렇게 표현할 수 있으면 어떨까 싶습니다!